libraries/bezier-rs/src/bezier/core.rs

```diff
@@ -218,6 +218,20 @@ impl Bezier {
 
 		self.get_points().all(|point| point.abs_diff_eq(start, MAX_ABSOLUTE_DIFFERENCE))
 	}
+
+	// TODO: What about the case when a handle extends beyond the start or end point? Is that considered "equivalent to a line"?
```

Comment:
I added this TODO comment. Can you please resolve this by mentioning the appropriate thing here to document this edge case? Thanks!

Reply:
Actually this was originally put because the pen tool uses cubic beziers for straight lines. The handle are exactly at the same position of start and end. Since its actual use-case was covered. I did not think any further. If we want it to work for all cases then I will require some time think about it.